python/ray/tests/test_resource_demand_scheduler.py

```diff
@@ -97,13 +102,13 @@ def test_get_instances_packing_heuristic():
         == [("m4.16xlarge", 1), ("m4.large", 1)]
     assert get_instances_for(
         TYPES_A, {}, 9999, [{"CPU": 64}, {"CPU": 9}, {"CPU": 9}]) == \
-        [("m4.16xlarge", 1), ("m4.4xlarge", 2)]
+        [("m4.16xlarge", 2)]
```

Comment:
Instead of updating the shared config for all the tests, can you add a new test config so that the existing tests are unchanged (and test the original behaviour)?